content/root/best-practice-guide.textile

```diff
@@ -58,7 +58,7 @@ The main factors are summarized here:
 
 * If you want to maintain a persistent connection with Ably's Realtime platform in order to subscribe to messages sent on a channel, or be notified whenever the presence set changes, you need to use the "realtime library":/realtime.
 * If you just want to publish data to the platform and don't need to subscribe or manage channel attachment, you can the "REST library":/rest.
-* If you want to publish at very high rates or with the lowest possible realtime latencies, "Ably's realtime library would be best as it can publish at higher rates and lower latencies than is possible with the REST library due to message pipelining":https://faqs.ably.com/should-i-use-the-rest-or-realtime-library.
+* If you want to publish at very high rates or with the lowest possible realtime latencies, Ably's realtime library would be best as it can publish at higher rates and lower latencies than is possible with the REST library due to message pipelining.
```

Comment:
This seems to remove the link, but I think we need to link to where we have this info in the main docs. For example:

* https://ably.com/documentation/realtime#realtime-vs-rest
* https://ably.com/documentation/rest#rest-vs-realtime
* https://ably.com/documentation/best-practice-guide#rest-or-realtime

Generally, we want to change links to KB/FAQ articles to link to the same info in the docs. Where that info is not currently in the main docs, we need to create new tickets to capture that.

As an aside, "pipelining" needs to be clarified, but that's probably beyond the scope of this PR and covered by [DOC-410](https://ably.atlassian.net/browse/DOC-410).

Reply:
I've updated it now to point to the [rest landing page](https://ably.com/documentation/realtime) as the other links just go to the relevant landing page and Mark made a good point that linking from Rest vs Realtime in one page to Rest vs Realtime in another felt a bit odd.